NebulaWorld/Logistics/StationUIManager.cs

```diff
@@ -0,0 +1,433 @@
+﻿using NebulaModel.Packets.Logistics;
+using HarmonyLib;
+using System;
+using System.Collections.Generic;
+using NebulaModel.DataStructures;
+using NebulaModel.Networking;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace NebulaWorld.Logistics
+{
+    public static class StationUIManager
+    {
+        private static ThreadSafeDictionary<int, List<NebulaConnection>> StationUISubscribers;
```

Comment:
This is gone in latest master.

Reply:
Im using the normal `Dictionary` there for now